src/main/java/org/apache/sysds/runtime/instructions/FEDInstructionParser.java

```diff
@@ -32,8 +36,28 @@
 	public static final HashMap<String, FEDType> String2FEDInstructionType;
 	static {
 		String2FEDInstructionType = new HashMap<>();
-		String2FEDInstructionType.put("fedinit", FEDType.Init);
-		String2FEDInstructionType.put("ba+*",    FEDType.AggregateBinary);
+		String2FEDInstructionType.put("fedinit"  , FEDType.Init);
+		String2FEDInstructionType.put("tsmm"     , FEDType.Tsmm);
+		String2FEDInstructionType.put("ba+*"     , FEDType.AggregateBinary);
+
+		String2FEDInstructionType.put("uak+"     , FEDType.AggregateUnary);
+		String2FEDInstructionType.put( "uark+"   , FEDType.AggregateUnary);
+		String2FEDInstructionType.put( "uack+"   , FEDType.AggregateUnary);
+		String2FEDInstructionType.put( "uasqk+"  , FEDType.AggregateUnary);
+		String2FEDInstructionType.put( "uarsqk+" , FEDType.AggregateUnary);
+		String2FEDInstructionType.put( "uacsqk+" , FEDType.AggregateUnary);
```

Comment:
inlining

Reply:
What do you mean?

Reply:
look at the code, some of the lines you insert spaces, some you don't.

Reply:
I added a new commit with the two spaces added. If you use an editor like IntelliJ, this sort of thing is aligned even if a single space is missing.